Header header
string ip
string descrioption 
string serial
string firmware_ver
uint8 fractional_bits

